robot_calibration/launch/joint_states_update.launch

```diff
@@ -4,4 +4,5 @@
     
    <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher"/>
    <node name="simple_joint_state_pub_1" pkg="robot_calibration" type="simple_joint_state_pub" />
-</launch>
\ No newline at end of file
+</launch>
```

Comment:
Add rviz to your launch file as well, with the correct config file specified. This can be done as follows:
```
<launch>
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find package_name)/rviz/config_file.rviz" />
</launch>
```
_Optional:_ Add a [command line arg](http://wiki.ros.org/roslaunch/Commandline%20Tools) to only launch rviz in debug mode ([docs](https://wiki.ros.org/roslaunch/XML) for conditions in roslaunch).